### PR TITLE
Blades: Model front end changes 

### PR DESCRIPTION
**Pull request by clairecfu**
- [x] added "Mount Type" to table and detail view 
- [x] added "Mount Type" to model form -> model form should be done for blades 
- [x] fixed bug where continued requests are made if no vendors exist 
- [x] fixed bug where model detail view crashes if no existing assets

### CODE REVIEW

File: src/utils/utils.tsx [hunk 1]
```diff
@@ -197,10 +200,17 @@ export interface DatacenterObject extends ElementObject {
   abbreviation: string;
 }
 
+export enum MountTypes {
+  RACKMOUNT = "Asset",
+  BLADE_CHASSIS = "Blade Chassis",
+  BLADE = "Blade Server ",
+}
+
```
Comment (clairecfu) [inline src/utils/utils.tsx:208]: this mirrors the strings on the backend. Even though i just changed these i am not super happy with these... after reading the requirements i think it should be "Normal Rackmount, Blade Chassis, Blade", which mirrors the vocabulary in the requirements

File: src/utils/utils.tsx [hunk 2]
```diff
@@ -115,6 +115,7 @@ export const ModelFieldsTable: any = {
   memory_gb: "Memory (GB)",
   storage: "Storage",
   comment: "Comment",
+  model_type: "Mount Type"
```
Comment (clairecfu) [inline src/utils/utils.tsx:118]: similarly, i think this should be called Mount Type, not model type, but keeping variable names consistent with backend for now

File: src/forms/modelForm.tsx [hunk 3]
```diff
@@ -189,163 +178,194 @@ class ModelForm extends React.Component<ModelFormProps, ModelFormState> {
...
+                value={values.display_color ? values.display_color : "#394B59"}
+                onChange={this.handleChange}
+              />
+            </FormGroup>
+          </Card>
+          <Card>
+            <RadioGroup
+              label="Mount Type*"
+              className="radio-group"
+              onChange={(e: any) => {
```
Comment (benauriemma) [inline src/forms/modelForm.tsx:248]: This is picky but I'm not sure Mount Type is accurate bc both classic assets and blade chassis are rackmount.  Maybe model type?  Or is there another word that describes it better

Comment (benauriemma) [reply src/forms/modelForm.tsx:248]: Discussed in standup that'll we'll leave it but ask Bletsch to confirm - are you gonna post on Piazza?

File: src/forms/modelForm.tsx [hunk 4]
```diff
@@ -189,163 +178,194 @@ class ModelForm extends React.Component<ModelFormProps, ModelFormState> {
...
+            >
+              <Radio
+                label={MountTypes.RACKMOUNT}
+                value={MountTypes.RACKMOUNT}
+              />
+              <Radio
+                label={MountTypes.BLADE_CHASSIS}
+                value={MountTypes.BLADE_CHASSIS}
+              />
+              <Radio label={MountTypes.BLADE} value={MountTypes.BLADE} />
```
Comment (benauriemma) [inline src/forms/modelForm.tsx:265]: My new least favorite thing the linter has done

File: src/utils/utils.tsx (hunk 2 above)
Comment (benauriemma) [reply src/utils/utils.tsx:118]: You're tracking changing this assuming Bletsch confirms?

Comment (clairecfu) [reply src/utils/utils.tsx:119]: yeet

File: src/forms/modelForm.tsx (hunk 4 above)
Comment (clairecfu) [reply src/forms/modelForm.tsx:265]: hehehehehehehe

File: src/forms/modelForm.tsx (hunk 3 above)
Comment (clairecfu) [reply src/forms/modelForm.tsx:248]: yep

File: src/utils/utils.tsx (hunk 1 above)
Comment (benauriemma) [reply src/utils/utils.tsx:209]: I assume you're tracking

**Pull request merged**